words/words.md

```diff
@@ -73,11 +73,11 @@ Wenn wir "eine Verdoppelung alle 4 Tage", bei einer Bevölkerung, die mit nur 0,
 		<iframe src="sim?stage=epi-1" width="800" height="540"></iframe>
 </div>
 
-Dies ist die **exponentielle Wachstumskurve.** Sie fängt klein an und explodiert dann. "Oh, es ist nur eine Grippe" bis "Oh richtig, Grippe schafft keine *Massengräber in reichen Städten*". 
+Dies ist die **exponentielle Wachstumskurve.** Sie fängt klein an und explodiert dann. Von "Ach, es ist nur eine Grippe" bis "Oh richtig, eine Grippe schafft keine *Massengräber in reichen Städten*". 
```

Comment:
Klingt eindeutig besser.

Reply:
Würde 
"Oh richtig, eine Grippe schafft keine *Massengräber in reichen Städten*""
evtl. ersetzen durch 
"Oh richtig, eine Grippe schafft legt keine *Massengräber in reichen Städten* an"

"Massengräber schaffen" klang für mich etwas seltsam. "Massengräber anlegen" scheint nach kurzer Recherche gängig. Einfach übernehmen wenn es sich gut anfühlt, sonst ist egal.

Reply:
find's so auch gut